lttoolbox/fst_processor.h

```diff
@@ -443,6 +438,19 @@ class FSTProcessor
   bool isLastBlankTM = false;
 
   xmlTextReaderPtr reader;
+
+  int max_case_insensitive_state_size = 65536;
```

Comment:
Should be `constexpr size_t`

Reply:
I got a warning on size_t since state.size returns int, but then perhaps state.size should be changed too?
https://github.com/apertium/lttoolbox/blob/167-avoid-already-applied-destinations/lttoolbox/state.cc#L84-L88

Reply:
Yes, since `vector.size()` is `size_t`. Could even make `State::size()` return `auto` so it deduces as whatever `vector.size()` returns.